src/commands/add.ts

```diff
@@ -463,20 +464,31 @@ export async function setInfo(message: Message, list: number, row: Row) {
 				if (tags?.length > 0) {
 					siteTags.tags = [...tags];
 				}
-				if (chars?.length > 0) {
-					siteTags.characters = [...chars];
+				
+				const chars2: string[] = [...chars]; //chars will be used in the detectedCharacters block
+				
+				if (chars2?.length > 0) {
+					for (let t = 0; t < chars2.length; t++) {
+						for (const [key, value] of Object.entries(renameCharacters)) {
```

Comment:
Don't iterate over this, a simple check for "if char2[t] in renameCharacters" will suffice